traveler-ios-ui/TravelerKitUI/ViewControllers/BookableQuestionsViewController.swift

```diff
@@ -1,5 +1,5 @@
 //
-//  BookingQuestionsViewController.swift
```

Comment:
Rename back to original

Reply:
Some of the VCs are called Bookable...VC and should we also call it Bookable instead of Booking?